solution-base/build.sh

```diff
@@ -0,0 +1,180 @@
+#!/bin/bash
+set -eu
+
+PWD=$(pwd)
+BUILD_ROOT=${PWD}/_build
+ISO_ROOT=${BUILD_ROOT}/root
+IMAGES_ROOT=${ISO_ROOT}/images
+
+PRODUCT_NAME=Zenko-Base
+PRODUCT_LOWERNAME=zenko-base
+BUILD_TIMESTAMP=$(date -u +"%Y-%m-%dT%H:%M:%SZ")
+BUILD_HOST=$(hostname)
+
+VERSION_SHORT=$(git describe --abbrev=0)
+VERSION_FULL=${VERSION_SHORT}-dev
+GIT_REVISION=$(git describe --long --always --tags --dirty)
+ISO=${BUILD_ROOT}/${PRODUCT_LOWERNAME}-${VERSION_FULL}.iso
+
+DOCKER=docker
+DOCKER_OPTS=
+DOCKER_SOCKET=${DOCKER_SOCKET:-unix:///var/run/docker.sock}
+HARDLINK=hardlink
+SKOPEO=skopeo
+SKOPEO_OPTS="--override-os linux --insecure-policy"
+OPERATOR_TAG=$(grep /operator: deps.txt | awk -F ':' '{print $2}')
+SOLUTION_REGISTRY=metalk8s-registry-from-config.invalid/${PRODUCT_LOWERNAME}-${VERSION_FULL}
+
+KUBEDB_SCRIPT_BRANCH_TAG=89fab34cf2f5d9e0bcc3c2d5b0f0599f94ff0dca
+
+export KUBEDB_OPERATOR_TAG=${OPERATOR_TAG}
+export KUBEDB_SCRIPT_LOCATION="curl -fsSL https://raw.githubusercontent.com/kubedb/installer/${KUBEDB_SCRIPT_BRANCH_TAG}/"
+export KUBEDB_NAMESPACE=SOLUTION_ENV
+export KUBEDB_SERVICE_ACCOUNT=kubedb-operator
+export KUBEDB_OPERATOR_NAME=operator
```

Comment:
```suggestion
export KUBEDB_OPERATOR_NAME=kubedb-operator
```

this would remove the need to prefix `kubedb-` in many parts in the yaml. 
(unless I missed something)

Reply:
Maybe, I simply copy pasted this from the kubedb operator yaml. I tried to make as few changes as possible.